quarkus/tests/junit5/src/main/java/org/keycloak/it/utils/DockerKeycloakDistribution.java

```diff
@@ -292,4 +341,8 @@ public void clearEnv() {
         this.envVars.clear();
     }
 
+    public int getMappedPort(int port) {
+        return keycloakContainer.getMappedPort(port);
```

Comment:
Does it make sense to add a null check? I can imagine a scenario, where KeycloakContainer could be null and subsequently getMappedPort could be called.